client/extensions/woocommerce/app/dashboard/manage-external-view.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+
+class ManageExternalView extends Component {
+	static propTypes = {
+		site: PropTypes.shape( {
+			URL: PropTypes.string.isRequired,
+		} ),
+	};
+
+	render = () => {
+		const { site, translate } = this.props;
+
+		const title = translate( 'Manage your store' );
+		const line = translate(
+			'Stores in your country are managed directly on your site. ' +
+				'We will let you know when you can manage your store here.'
+		);
+		const actionURL = site.URL + '/wp-admin/edit.php?post_type=shop_order';
```

Comment:
If we end up going this route, I'd love to have some tracks on the number of clicks here.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/21466/commits/e88080314c755645ae2dff6a65342babff2b7d5f